helm/alfresco-content-services/values.yaml

```diff
@@ -79,7 +79,7 @@ repository:
       memory: "3000Mi"
   # -- The startup probe to cover the worse case startup time for slow start containers
   startupProbe:
-    periodSeconds: 30
+    periodSeconds: 180
```

Comment:
That is probably only valid for your env that is slow. We don't want that as a default for prod env

Reply:
Fixed